tests/lora/test_multilora_hpu.py

```diff
@@ -0,0 +1,134 @@
+from multiprocessing import Process
+from typing import List, Optional, Tuple
+
+from vllm import EngineArgs, LLMEngine, RequestOutput, SamplingParams
+from vllm.lora.request import LoRARequest
+
+
+def create_test_prompts(
+        lora_path: str
+) -> List[Tuple[str, SamplingParams, Optional[LoRARequest]]]:
+    """Create a list of test prompts with their sampling parameters.
+
+    2 requests for base model, 4 requests for the LoRA. We define 2
+    different LoRA adapters (using the same model for demo purposes).
+    Since we also set `max_loras=1`, the expectation is that the requests
+    with the second LoRA adapter will be ran after all requests with the
+    first adapter have finished.
+    """
+    # TODO Fix issues when enabling parameters [presence_penalty=0.2,
```

Comment:
presence_penalty issue is fixed on latest. Please test with LoRA and restore this configuration.

Reply:
Done